cli/ioctl/cmd/account/accountimport.go

```diff
@@ -85,15 +83,11 @@ func writeToFile(alias, addr string) (string, error) {
 		alias), nil
 }
 func readPasswordFromStdin() (string, error) {
```

Comment:
this is a misleading function name. What it is intended to do is to ready private key, but not password. This is the reason spaces are trimmed. Moreover, this is a wrong logic, if the private key length is not valid, we are supposed to reject the private key. Let's keep it as it is for now, and fix the problem in another PR.

Reply:
For readPasswordFromStdin in line 119, you can replace it with the function you write "util.TypePassword()"